board/samsung/board-j4lte.c

```diff
@@ -0,0 +1,8 @@
+/* SPDX-License-Identifier: GPL-2.0 */
+/*
+ * Copyright (c) 2022
```

Comment:
Add yourself to the copyright or you could ask ivoszbg if it's fine to leave out (or maybe add Linux foundation?)

Reply:
okay